doc/source/tutorial/linalg_batch.md

```diff
@@ -0,0 +1,148 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.4
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+orphan: true
+---
+
```

Comment:
Should this have? I saw it in the other notebooks you added.
```suggestion
```{eval-rst}
.. jupyterlite:: ../../_contents/hypothesis_shapiro.ipynb
   :new_tab: True
```
```

Reply:
This is the way to make the notebook executable and have the "Open in notebook" button appear, yes. If that is desired here, feel free to add - the ipynb file is already created as part of the "convert-notebooks" script in the Makefile.